Library/Formula/consul.rb

```diff
@@ -13,6 +13,8 @@ class Consul < Formula
     sha256 "67052157266d536598ba7eb5af31a10fd3e609ca812be6d50869d0fe80a583f9" => :mountain_lion
   end
 
+  option "with-web-ui", "Installs the consul web ui (to /usr/local/share/consul directory)"
```

Comment:
I just wanted to give them that additional bit of info so that they don't have to read the formula to figure it out. Is it ok I say something like this "<prefex>/share/consul"?